integration-cli/docker_cli_ps_test.go

```diff
@@ -304,6 +305,8 @@ func TestPsListContainersFilterStatus(t *testing.T) {
 	// FIXME: this should test paused, but it makes things hang and its wonky
 	// this is because paused containers can't be controlled by signals
 	deleteAllContainers()
```

Comment:
But that `deleteAllContainers` a hack. And it used only for this test, though another `Ps` tests is the same vulnerable to leaked containers, so I don't see a point for keeping it.